src/content/1.7/development/architecture/migration-guide/testing/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Testing
+menuTitle: Testing
+---
+
+# Testing
+
+You are encouraged to add both unit and functional tests for every new class
```

Comment:
@Progi1984 will ❤️

Reply:
haha, I just moved it from another file. (actually I think we are not adding smoke tests for every class :smile:)